tensorflow/core/kernels/mkl_dequantize_op_test.cc

```diff
@@ -59,4 +61,83 @@ TEST_F(MklDequantizeOpTest, small) {
   test::ExpectTensorNear<float>(expected, output, 0.1);
 }
 
+Tensor CreateMklInput() {
+  MklDnnShape mkl_shape;
+  memory::desc md =
+      memory::desc({1, 2, 2, 2}, MklDnnType<uint8>(), memory::format::nhwc);
+  mkl_shape.SetMklTensor(true);
+  mkl_shape.SetMklLayout(&md);
+  mkl_shape.SetElemType(MklDnnType<uint8>());
+  mkl_shape.SetTfLayout(4, {1, 2, 2, 2}, memory::format::nhwc);
+
+  DataType dtype = DataTypeToEnum<uint8>::v();
+  Tensor mkl_tensor(dtype, {mkl_shape.GetSerializeBufferSize()});
+  mkl_shape.SerializeMklDnnShape(
+      mkl_tensor.flat<uint8>().data(),
+      mkl_tensor.flat<uint8>().size() * sizeof(uint8));
+  return mkl_tensor;
+}
+
+template <typename T>
+class CommonTestUtilities : public OpsTestBase {
+ public:
+  void MklToTF(const Tensor& tensor, const Tensor& mkl_meta_tensor,
+               Tensor* output) {
+    // Create an MKL to TF conversion node and execute it
+    TF_ASSERT_OK(NodeDefBuilder("mkl_to_tf_op", "_MklToTf")
+                     .Input(FakeInput(DataTypeToEnum<T>::v()))
+                     .Input(FakeInput(DT_UINT8))  // MKL second tensor
+                     .Attr("T", DataTypeToEnum<T>::v())
+                     .Attr("_kernel", "MklLayoutDependentOp")
+                     .Finalize(node_def()));
+    TF_ASSERT_OK(InitOp());
+    AddInputFromArray<T>(tensor.shape(), tensor.flat<T>());
+    AddInputFromArray<uint8>(mkl_meta_tensor.shape(),
+                             mkl_meta_tensor.flat<uint8>());
+    TF_ASSERT_OK(RunOpKernel());
+
+    *output = *GetOutput(0);
+  }
```

Comment:
I remember seeing this in many test files. Can we refactor this to a common test utility file in a future PR? Thank you!

Reply:
@penpornk sure, we will move this general code to a util file in the feature.